### PR TITLE
v7.0 Notice

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by itao — APPROVED** [review]

File: notices.json [hunk 1]
```diff
@@ -18,21 +18,21 @@
...
       "instanceType": "onprem",
-      "displayDate": ">= 2022-05-18T00:00:00Z"
+      "displayDate": ">= 2022-06-20T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
-        "title": "Mattermost 6.7 is here!",
-        "description": "Mattermost v6.7 includes [Playbooks task due dates](https://docs.mattermost.com/playbooks/customize-a-playbook.html#assign-tasks-and-due-dates). [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
+        "title": "Mattermost 7.0 is here!",
+        "description": "Mattermost v7.0 includes general availability of [collapsed reply threads](https://docs.mattermost.com/channels/organize-conversations.html), [voice calls and screen share (beta)](https://docs.mattermost.com/channels/make-calls.html), messaging format toolbar, [Playbooks inline editor](https://docs.mattermost.com/playbooks/customize-a-playbook.html#edit-a-playbook), usage statistics, and ability to change [triggers and actions mid-run](https://docs.mattermost.com/playbooks/customize-a-run.html). [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
```
Comment (esethna) [inline notices.json:32]: [messaging formatting toolbar](https://docs.mattermost.com/channels/format-messages.html)

**Review by esethna — APPROVED** [review]
Looks good with one tweak

**Review by esethna — APPROVED** [review]

**Comment by ogi-m** [comment]
LGTM!

image

**Review by ogi-m — APPROVED** [review]

**Comment by amyblais** [comment]
@esethna @ogi-m Sorry, I need one approval to be able to merge this. The PR checks seemed to be stuck and I made a minor commit and the previous approvals got dismissed.

**Review by ogi-m — APPROVED** [review]

**Comment by ogi-m** [comment]
@amyblais Done!

**Comment by amyblais** [comment]
Thank you!

**Pull request merged**